src/main/kotlin/com/sourcegraph/cody/config/SettingsMigration.kt

```diff
@@ -44,6 +45,9 @@ class SettingsMigration : Activity {
     RunOnceUtil.runOnceForProject(project, "CodyConvertUrlToCodebaseName") {
       migrateUrlsToCodebaseNames(project)
     }
+    RunOnceUtil.runOnceForProject(project, "CodyAccountHistoryMigration") {
```

Comment:
Shouldn't that migration be run as last one?

Reply:
This order seems to be good.